src/pymmcore_widgets/hcs/_main_wizard/_main_wizard.py

```diff
@@ -0,0 +1,289 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from pymmcore_plus import CMMCorePlus
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtWidgets import QFileDialog, QVBoxLayout, QWidget, QWizard
+from superqt.utils import signals_blocked
+from useq import Position, RandomPoints, WellPlatePlan
+
+from pymmcore_widgets.hcs._calibration_widget._calibration_widget import (
+    CalibrationData,
+)
+from pymmcore_widgets.hcs._fov_widget._fov_sub_widgets import Center
+from pymmcore_widgets.hcs._graphics_items import Well
+from pymmcore_widgets.hcs._plate_widget import PlateInfo
+from pymmcore_widgets.hcs._util import (
+    DEFAULT_PLATE_DB_PATH,
+)
+
+from ._main_wizard_pages import FOVSelectorPage, PlateCalibrationPage, PlatePage
+
+if TYPE_CHECKING:
+    from useq import WellPlate
+
+
+class HCSWizard(QWizard):
+    """A wizard to setup an High Content experiment.
+
+    This widget can be used to select a plate, calibrate it, and then select the FOVs
+    to image in different modes (Center, RandomPoint or GridRowsColumns).
+
+    Parameters
+    ----------
+    parent : QWidget | None
+        The parent widget. By default, None.
+    mmcore : CMMCorePlus | None
+        The CMMCorePlus instance. By default, None.
+    plate_database_path : Path | str | None
+        The path to the plate database. By default, None.
```

Comment:
in this PR, I would get rid of DEFAULT_PLATE_DB_PATH, any public facing API about databases, and probably even the database files themselves. most of that can go into useq-schema. (the main thing I don't want in useq-schema is the concept of a user-editable database that we load from file).

I'd prefer to leave the entire discussion of any public-facing concept of a user database to another PR. Feel free to either add typical defaults to useq-schema, or call useq.register_well_plate (or whatever it was) with some additional data here, but let's not (yet) propagate this concept of a "database" all over this PR